src/language-css/clean.js

```diff
@@ -9,6 +9,7 @@ const ignoredProperties = new Set([
   "before",
   "after",
   "trailingComma",
+  "spaces",
```

Comment:
This is an object, was treat as node before, and removed `.before`, `.after`.  It's not a node and only have `before`/`after`, we can just ignore it.